tests/test_e2e.py

```diff
@@ -83,7 +82,7 @@ def test_e2e(tmp_path):
 
     # This df contains data w/ two ITRFs: ITRF2005 and ITRF2008.
     complete_df = pd.concat(all_dfs)
-    complete_df = DataFrame_co[atm1bData](complete_df)
+    complete_df = ATM1BData(complete_df)
```

Comment:
This looks more readable/understandable than what it was changed from, in my eyes. Maybe I'm just not used to using `TypeVar`.

Reply:
You can give `DataFrame_co[atm1bData]` an alias. But I'm wondering now, why use both TypeVar _and_ PEP695 generic syntax? I thought PEP695 was ruled out (similar comment in #8)

Reply:
Hmm, an alias might be a nice way to handle this.

Tbh I'm still pretty fuzzy on PEP695 and how this is working. T

Reply:
My fault, I'm being confused and confusing. Please ignore my comment on PEP695 here.

Reply:
Not to worry - I'm confused too :rofl: